airflow/example_dags/example_latest_only_decorator.py

```diff
@@ -0,0 +1,58 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Example of the `latest_only` decorator"""
+
+from airflow import DAG
+from airflow.models.baseoperator import BaseOperator
+from airflow.operators.bash import BashOperator
+from airflow.utils.dates import days_ago
+from airflow.utils.decorators import latest_only
+
+
+# [START example]
+class MyLatestOnlyOperator(BaseOperator):  # pylint: disable=missing-class-docstring
+    def __init__(self, latest_only=True, **kwargs):  # pylint: disable=redefined-outer-name
+        self.latest_only = latest_only
+        if self.latest_only:
+            self.message = 'I will skip all but the latest execution'
+        else:
+            self.message = 'I will never skip'
+        super().__init__(**kwargs)
+
+    @latest_only
+    def execute(self, context):
+        print(self.message)
+
+
+class MyBashOnlyOperator(BashOperator):  # pylint: disable=missing-class-docstring
+    @latest_only
+    def execute(self, context):
+        print('I will skip all but the latest execution')
+        super().execute(context=context)
```

Comment:
It would be nice if this could work as

```
     subclass_example = latest_only(BashOperator(task_id='latest_only_bash_task', bash_command='echo "hello world"'))
```

or similar, but I'm not sure if that is realistically achievable.

Reply:
you know what, maybe latest only should just be added as a parameter to base operator?  that would seem to fit with airflow style more so than wrapping in this function. wdyt?
so that every operator has this `latest_only` parametr

Reply:
I like the idea to be just `latest_only ` param, and it allows all operators to benefit from it right away!

Reply:
okie doke, i have added it to base operator, ptal 🙏